src/subcategories/id.ts

```diff
@@ -1,7 +1,23 @@
 import { makeEnum } from '@transcend-io/type-utils';
 
+/** Information that can be used to identify an individual */
 export const IdSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** A username associated with an individual */
+  Username: 'USERNAME',
+  /** An individual's email address */
+  Email: 'EMAIL',
```

Comment:
I think there is already another entry under the `ContactSubCategory` enum with the exact same key and value. All of the subcategory enums are combined into one [here](https://github.com/transcend-io/privacy-types/blob/3ff1c251c4bfb823947a214b515449b56bca72f2/src/subcategories/index.ts#L18) so probably need to have a different key/value for this.

Reply:
good catch - i'll just remove this one